pyrovision/datasets/openfire.py

```diff
@@ -20,179 +19,125 @@ class OpenFire(VisionDataset):
     """Wildfire image Dataset.
 
     Args:
-        root (string): Root directory of dataset where the ``images``
-            and  ``annotations`` folders exist.
+        root (string): Root directory of dataset.
         train (bool, optional): If True, returns training subset, else test set.
+        transform (callable, optional): A function/transform that  takes in an PIL image
+            and returns a transformed version. E.g, ``transforms.RandomCrop``
+        target_transform (callable, optional): A function/transform that takes in the
+            target and transforms it.
         download (bool, optional): If true, downloads the dataset from the internet and
             puts it in root directory. If dataset is already downloaded, it is not
             downloaded again.
-        threads (int, optional): If download is set to True, use this amount of threads
-            for downloading the dataset.
-        num_samples (int, optional): Number of samples to download (all by default)
-        img_folder (str or Path, optional): Location of image folder. Default: <root>/OpenFire/images
-        **kwargs: optional arguments of torchvision.datasets.VisionDataset
+        sample (bool, optional): If True, use openfire subset with 64 training images and 16 testing.
     """
 
-    url = 'https://gist.githubusercontent.com/frgfm/f53b4f53a1b2dc3bb4f18c006a32ec0d/raw/c0351134e333710c6ce0c631af5198e109ed7a92/openfire_binary.json'  # noqa: E501
-    classes = [False, True]
-
-    def __init__(self, root, train=True, download=False, threads=None, num_samples=None,
-                 img_folder=None, **kwargs):
-        super(OpenFire, self).__init__(root, **kwargs)
-        self.train = train
-        if img_folder is None:
-            self.img_folder = Path(self.root, self.__class__.__name__, 'images')
-        else:
-            self.img_folder = Path(img_folder)
+    urls = {'ds': 'https://github.com/pyronear/pyro-vision/releases/download/v0.1.2/open_fire.zip',
+            'sample': 'https://github.com/pyronear/pyro-vision/releases/download/v0.1.2/open_fire_sample.zip'
+            }
+
+    md5s = {'ds': '5a532853ac17dc43ed7dd4a97a15d715',
+            'sample': '31117230cceb029d557a1981f0f30cf7'
+            }
+
+    classes = ['0 - No Fire', '1 - Fire']
+
+    def __init__(
+            self,
+            root: str,
+            train: bool = True,
```

Comment:
There is no difference whatever the value of `train`. Perhaps we should remove this

Reply:
there is a difference [here](https://github.com/MateoLostanlen/pyro-vision/blob/2a71cc6a7e2ace51f7ffa3d472d91466fa9d2973/pyrovision/datasets/openfire.py#L72)